node-rti/src/ext.rs

```diff
@@ -51,6 +52,40 @@ impl ProgramStorage for ExtProgramStorage {
     }
 }
 
+impl ExtProgramStorage {
+    pub fn iter(&self) -> ExtProgramStorageIter {
+        ExtProgramStorageIter {
+            key: Some(b"g::prog".to_vec()),
```

Comment:
It is worth replacing `b"g::prog"`with the named const.

Reply:
Decided to declare const for all fixed prefixes and keys in `gear-common`, done